tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/appsettings.json

```diff
@@ -40,10 +40,7 @@
     "StampedPropertyResponsibleAttorney": "ResponsibleAttorney",
     "StampedPropertyTeamMembers": "TeamMembers",
     "StampedPropertyBlockedUploadUsers": "BlockedUploadUsers",
-    "StampedPropertyMatterCenterUserEmails": "MatterCenterUserEmails",
```

Comment:
These stamped properties are not useful. These are moved to ContentType Section as part of ManagedColumns and will be read from there